R/GeospatialFunctions.R

```diff
@@ -44,7 +44,7 @@ TADA_MakeSpatial <- function(.data, crs = 4326) {
       "NAD83", 4269,
       "WGS84", 4326,
       "NAD27", 4267,
-      "UNKWN", crs, # Unknowns and NAs should go to user supplied default
+      "UNKWN", crs,  # Unknowns and NAs should go to user supplied crs or default
       "OTHER", 4326,
```

Comment:
I've run into this one as well

```suggestion
      "Unknown", crs,
      "OTHER", 4326,
```

Reply:
Should "OTHER" also be user supplied conversion?

Reply:
I concur.... though I also think it would be fine to not let this be a user-defined argument and coerce all to 4326 or 4269.